web/src/components.ts

```diff
@@ -115,10 +116,13 @@ export function toggle(opts: {
         return false;
     }
 
-    meta.$ind_tab.on("click", function () {
-        const idx = Number($(this).attr("data-tab-id"));
-        select_tab(idx);
-    });
+    function register_event_handlers(): void {
+        meta.$ind_tab.on("click", function () {
+            const idx = Number($(this).attr("data-tab-id"));
+            select_tab(idx);
+        });
```

Comment:
This doesn't remove any existing event handlers; so we need to be careful using this API to avoid letting multiple get applied to the target element. Do we need an `off` removal first?